cumulusci/robotframework/CumulusCI.py

```diff
@@ -116,10 +116,17 @@ def login_url(self, org=None, **userfields):
 
         """
         org = self.org if org is None else self.keychain.get_org(org)
+
         if userfields:
-            access_token = org.get_access_token(**userfields)
-            login_url = f"{org.instance_url}/secur/frontdoor.jsp?sid={access_token}"
-            return login_url
+            if org.get_access_token is not None:
+                # connected persistent org configs won't have the get_access_token method
+                access_token = org.get_access_token(**userfields)
+                login_url = f"{org.instance_url}/secur/frontdoor.jsp?sid={access_token}"
```

Comment:
Can these two lines be merged with the same ones in the next branch?

Reply:
I've refactored this section slightly to move the common code outside of the `if` statements.